stations.csv

```diff
@@ -32263,3 +32263,6 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 35088;Villa Convento;villa-convento;;;40.36534;18.069935;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;8341377;t;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 35089;Vitigliano;vitigliano;;;40.039633;18.412861;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;8341378;t;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 35090;Zollino Viale Italia;zollino-viale-italia;;;40.20528;18.246746;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;8341437;t;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+35091;Colleville-sur-Mer;colleville-sur-mer;;;49.2054;0.5037;;FR;Europe/Paris;f;f;f;t;f;f;FRUXH;;t;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+35092;Arromanches-les-Bains;arromanches-les-bains;;;49.2021;0.3721;;FR;Europe/Paris;f;f;f;t;f;f;FRUXG;;t;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```

Comment:
Arromanches-les-Bains' UIC code is `87701789`

Reply:
Where can we found this code?